nodes/src/nodes/tool_v0/v0_driver.py

```
@@ -0,0 +1,310 @@
+
+    # ------------------------------------------------------------------
+    # Tool implementations
+    # ------------------------------------------------------------------
+
+    def _build_headers(self) -> Dict[str, str]:
+        return {
+            'Authorization': f'Bearer {self._apikey}',
+            'Content-Type': 'application/json',
+        }
+
+    def _call_v0_api(self, messages: List[Dict[str, str]], model: str, **extra: Any) -> Dict[str, Any]:
```

Ruff (0.15.7)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**extra`

(ANN401)